csrc/logical_domain_map.cpp

```diff
@@ -263,7 +263,9 @@ std::unordered_map<IterDomain*, IterDomain*> PairwiseLogicalDomainMap::map(
     //   logsumexp = [DIDx(D)?, N, H, L]
     // Note: S, E are not mapped together in the producers and do not have any
     // mapping to the consumer.
-
+    if (consumer_tv_->sameAs(op->philox_seed())) {
```

Comment:
Should we also return {} if consumer_tv_ is op->philox_offset()?

Reply:
That is always 0-dim, so I didn't add it. I can add that if it seems more clear.